plugins/org.jboss.tools.livereload.ui/src/org/jboss/tools/livereload/ui/internal/command/OpenInWebBrowserViaLiveReloadProxyEnablementTester.java

```diff
@@ -34,7 +35,10 @@ public boolean test(Object receiver, String property, Object[] args, Object expe
 			return false;
 		}
 		final IServer appServer = appModule.getServer();
-		return appServer != null && WSTUtils.isServerStarted(appServer);
+		IModule[] modules = appModule.getModule();
+		return appServer != null
+				&& WSTUtils.isServerStarted(appServer)
+				&& WSTUtils.getModuleURL(appServer, modules[modules.length - 1]) != null;
 	}
```

Comment:
since you're adding more conditions to enable the command (which seems legit), could you add a set of tests to verify that it works as expected in all situations ? Eg: when the project is a .war or an .ear with web modules and ejb modules, started or stopped.